### ISSUE TITLE
The Feature of "Always Show Structure of Master Document" in Side Panel is Always Disabled by TeXStudio

### ISSUE THREAD

**Issue by qyua3175** [texstudio-org/texstudio#1829]
The Feature of "Always Show Structure of Master Document" in Side Panel allows users to facilitate the management of the document structure and continuously worked very well in all old versions. But this feature is always auto-disabled by the TeXStudio itself in the latest v4.0.0 Qt6. Users cannot see the structure tree when each session is initially opened, but have to firstly click the TOC button in the side panel, then the Structure Tree can be enabled.
This is not a major functional bug, but it will do impact the user experience and cause some inconvenience to authors. So please fix this bug asap.

**Comment by sunderme** [comment]
you can use globalTOC

but I look into it.

**Comment by qyua3175** [comment]
> you can use globalTOC
> 
> but I look into it.

Yes, I know the global TOC is normal for back up at this moment, but it also confirms the bug do exist. I want the developer team  to be notified and can fix it. Thanks for your efforts.

**Comment by sunderme** [comment]
fixed
